web/client/components/data/featuregrid/FeatureEditorFallback.jsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import ReactDock from 'react-dock';
```

Comment:
This fallback component is imported in a plugin where we are trying to reduce the entry point bundle size as much as possible so I would avoid to add a library for a fallback component.
Could we use only a div with a centered spinner?

please remove also other related functions for the dock

Reply:
Done